kits/arm/ex_teach_repeat_w_gripper.py

```diff
@@ -69,8 +70,22 @@
 B8 - Quit
 """
 print(instructions)
-m.clear_text()
-m.add_text(instructions)
+
+m.set_button_label(1, 'Add waypoint (Stop)')
+m.set_button_label(2, 'Add Waypoint (Stop) & Toggle Gripper')
```

Comment:
In particular, I think labels such as this are way too big for the buttons (at least on older mobile IO_.  This is why we tried to simplify to unicode symbols where possible, and let the more verbose instructions be covered in the text box/terminal.